### DISCUSSION TITLE
Add allocatable resources property

### DISCUSSION THREAD

**Discussion by ariguillegp** [rad-security/kbom#81] [Ideas]
Hi! First of all, great work with this initiative. I think there is a ton of value on creating a k8s focused BOM that could be used in the future as a data source for multiple applications in domains like SRE, Security, etc. I was looking at the output of the `generate` command, specifically the `nodes` key and I think it could be useful to also introduce the `allocatable` resources, which gives a more clear idea of the actual state of your cluster's nodes, allowing you to answer things like: what else could I schedule in these nodes or why is this node this saturated if only these tiny workloads are running there (potential attack where someone is running containers that were scheduled out of band). I'm curious about your take on this idea and I'd be more than happy to submit a PR if you think that's something that will add value to the report. Thanks!

**Comment by mateuszdyminski** [comment]
Hi @ariguillegp 

We like the idea to add extra info under the `nodes` key. Any contributions are welcome so feel free to implement `allocatable` :)

**Reply by ariguillegp** [reply]
here goes the [PR](https://github.com/ksoclabs/kbom/pull/82), thanks!

**Comment by mateuszdyminski** [comment]
Hi @ariguillegp,

Just merged your PR, thanks for contributing to the `kbom` tool! We really appreciate the help from the community!

Your changes are in our newest release: https://github.com/ksoclabs/kbom/releases/tag/v0.2.2

Cheers!
